.editorconfig

```diff
@@ -4,7 +4,7 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 4
+indent_size = 2
```

Comment:
This is my mistake to change the global value. If we don't need that, could you add something like this or I can create a PR to your branch

```
[*.md]
trim_trailing_whitespace = false
indent_size = 2
```